### ISSUE TITLE
Ollama API documentation should contain the default values for API options.

### ISSUE THREAD

**Issue by MarkWard0110** [ollama/ollama#4937]
As a developer, I want to know the default values Ollama uses for API request `options` so that I can create reproducible API calls between different kinds of clients.

Some AI libraries provide default values for options that are different from those used by the Ollama API. I want to override the library's default value with the Ollama default value.
I understand I will also need to check the `modelfile`, but if the `modelfile` does not specify a value, it will use what Ollama uses.

The https://github.com/ollama/ollama/blob/main/docs/modelfile.md#valid-parameters-and-values lists the options but with an example, which I don't assume may not represent the default value.

**Comment by jlawrence-yellostudio** [comment]
I agree it could be easier. There are some default values in that list at the end of the descriptions. However i'm not seeing all the same options in api.md to whats in modelfile.md.

FYI I did find a bunch of what i assume / believe to be defaults here:
https://github.com/ollama/ollama/blob/a6cd8f6169c029c92105962017562274bd90626b/api/types.go#L576
